### ISSUE TITLE
Improve progress bar UI experience

### ISSUE THREAD

**Issue by DuMaM** [jenkinsci/design-library-plugin#92]
### What feature do you want to see added?

Default progress bar in any modern web browser is prettier then current default one included in Jenkins.
![image](https://user-images.githubusercontent.com/9051964/180657724-2d3663a9-7690-4d74-8ad7-6073ea5ad5ae.png)

versus default one from Brave browser.
![image](https://user-images.githubusercontent.com/9051964/180657605-be6d15bf-77cd-4040-b72a-0de2b7fe8143.png)

Nowadays Jenkins is getting nicer and nicer UI, but progress bar didn't changed for ages and it's still uses tables under the hood.
https://github.com/jenkinsci/jenkins/blob/237b9ca457c032885ed98e29620f0ed8c0e55ca0/war/src/main/less/base/style.less#L1115

I thing implementing something more modern one (like one included bellow), can improve overall user experience.
https://css-tricks.com/css3-progress-bars/
![image](https://user-images.githubusercontent.com/9051964/180657818-673efa2c-85ea-4824-b6e0-9b38182f5879.png)

New progress bar will reduce problem like bellow ticket.
https://issues.jenkins.io/browse/JENKINS-68672?jql=text%20~%20progress%20AND%20text%20~%20bar%20ORDER%20BY%20created%20DESC

I will be really glad if such improvement will be implemented 😉

**Comment by NotMyFault** [comment]
Hey,

do you mind raising an enhancement on Jira with the details? The design library provides examples of UI related components and how to use them in plugins, it doesn't add or change anything on Jenkins itself.

**Comment by daniel-beck** [comment]
> New progress bar will reduce problem like bellow ticket

How? Your own screenshot shows a progress bar clearly indicating total width. Is that the progress bar you meant to reference?

>  https://github.com/frankie139506/jenkins/blob/3520a44fcd3957181bb11a02690038617eb0e945/war/src/main/less/base/style.less

Linking to a 1500+ line less file on a forked repo isn't useful.

**Comment by DuMaM** [comment]
> > New progress bar will reduce problem like bellow ticket
> 
> How? Your own screenshot shows a progress bar clearly indicating total width. Is that the progress bar you meant to reference?

I shared a search filter instead of specific ticket. Here you have example one.
https://issues.jenkins.io/browse/JENKINS-68672

For me there are two problems right now:
- accessibility 
- unmatched design

This screenshots was meant to be used as a reference point. I'm not a web programmer so I may be using a wrong terms here, but I will try to explain my idea as clear as possible.

Most of current UI changes should improve ux accessibility.
Current progress bar have one color used for filling without any frame what can serve as middle layer between background. 
Great example here is a current click effect in buttons for example (which I love btw). 
https://weekly.ci.jenkins.io/design-library/Buttons/

Those buttons add nice feedback for user. Even if they can have color blind problems, there is nice animation which tells that something got changed.

In terms of UI:
- no rounded eadges
- colors of progress bar do not match status icons

>
> > https://github.com/frankie139506/jenkins/blob/3520a44fcd3957181bb11a02690038617eb0e945/war/src/main/less/base/style.less
> 
> Linking to a 1500+ line less file on a forked repo isn't useful.

My bad. I didn't noticed this. 
I've already fixed description to line with code I think refers to this particular progress bar.
https://github.com/jenkinsci/jenkins/blob/237b9ca457c032885ed98e29620f0ed8c0e55ca0/war/src/main/less/base/style.less#L1115



> Hey,
> 
> do you mind raising an enhancement on Jira with the details? The design library provides examples of UI related components > and how to use them in plugins, it doesn't add or change anything on Jenkins itself.

Sure. I will add @daniel-beck suggestions there and add link to this issue as reference.
